spec/vcr_cassettes/ManageIQ_Providers_CiscoIntersight_PhysicalInfraManager_Refresher/refresh/will_perform_a_full_refresh.yml

```
@@ -33778,6 +33778,185 @@ http_interactions:
+        host date digest",signature="MEUCIGVj0xfRjnBm7x132CdrcorbNT8cELNV4D5L2TaDpJJ2AiEAl5RZugH4ZGB2E4Lo2ZTINN4qKBG3flG6xlLQlVXBxpw="
+      Date:
+      - Thu, 12 May 2022 13:46:56 GMT
+      Digest:
+      - SHA-256=47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=
+      Expect:
+      - ''
+  response:
+    status:
+      code: 200
+      message: ''
+    headers:
+      Content-Type:
+      - application/json; charset=utf-8
+      Date:
+      - Thu, 12 May 2022 13:46:56 GMT
+      Set-Cookie:
+      - AWSALB=XkWIj6bwzgQT58i+G6ozDEL426EyDG/BrktyWogB0N36WKVxE8pNxJuw168cByJy5GrVUPljRaI8wWqUFrPP4v5rsSxVB/tLWN8nZx7VsRw6HpbBz8X8jIrgRh8f;
+        Expires=Thu, 19 May 2022 13:46:56 GMT; Path=/
+      - AWSALBCORS=XkWIj6bwzgQT58i+G6ozDEL426EyDG/BrktyWogB0N36WKVxE8pNxJuw168cByJy5GrVUPljRaI8wWqUFrPP4v5rsSxVB/tLWN8nZx7VsRw6HpbBz8X8jIrgRh8f;
+        Expires=Thu, 19 May 2022 13:46:56 GMT; Path=/; SameSite=None; Secure
+      Server:
+      - nginx
+      X-Starship-Traceid:
+      - NBca08f754fa54715bb84593919f2fd61a
+      Pragma:
+      - no-cache
+      Cache-Control:
+      - no-cache, no-store, must-revalidate
+      Content-Security-Policy:
+      - 'base-uri ''self'' https://cdn.intersight.com/; child-src ''self'' https://www.intersight.com/
+        https://cdn.intersight.com/  https://cloudsso.cisco.com https://www.cisco.com/
+        https://id.cisco.com; default-src ''self'' https://cdn.intersight.com/; font-src
+        ''self'' data: https://cdn.intersight.com/; frame-ancestors ''self''; img-src
+        ''self'' data: https://cdn.intersight.com/; style-src ''self'' ''unsafe-inline''
+        https://cdn.intersight.com/; object-src ''none''; form-action ''self'' https://www.intersight.com/
+        https://cloudsso.cisco.com https://www.cisco.com/ https://id.cisco.com; script-src
+        ''self'' https://cdn.intersight.com/; sandbox allow-modals allow-scripts allow-same-origin
+        allow-popups allow-forms allow-downloads; connect-src ''self'' https://cdn.intersight.com/
+        https://status.intersight.com https://download.intersight.com/ wss://socket.intersight.com
+        wss://intersight.com; frame-src ''self'' blob: https://www.intersight.com/
+        https://cdn.intersight.com/  https://cloudsso.cisco.com https://www.cisco.com/
+        https://id.cisco.com;'
+      X-Xss-Protection:
+      - 1; mode=block;
+      Strict-Transport-Security:
+      - max-age=63072000; includeSubDomains
+      X-Content-Type-Options:
+      - nosniff
+      X-Frame-Options:
+      - SAMEORIGIN
+      X-Cache:
+      - Miss from cloudfront
+      Via:
+      - 1.1 158b0f42a1d87ab9108e2baf28e8c990.cloudfront.net (CloudFront)
+      X-Amz-Cf-Pop:
+      - ZAG50-C1
+      X-Amz-Cf-Id:
+      - zj3DxkQ3pJsUdc9vnh_OhYUxtoUzHgKINKUwpvtlxSYlGMppiwehkQ==
+    body:
+      encoding: ASCII-8BIT
+      string: |-
+            {
+        "ObjectType": "server.ProfileTemplate.List",
+        "Results":
+        [
+          {
```

```suggestion
        {
          "ObjectType": "server.Profile.List",
          "Results": [
            {
```

Then indent the rest of this accordingly